disttask/loaddata/proto.go

```diff
@@ -106,3 +119,26 @@ type FileInfo struct {
 	Size     int64
 	RealSize int64
 }
+
+// Mode records the mode information.
+type Mode struct {
+	Type     string
+	Logical  Logical
+	Physical Physical
+}
+
+// Logical records the logical mode information.
+type Logical struct{}
+
+// Physical records the physical mode information.
+type Physical struct {
+	RowIDBase int64
+	Timestamp int64
+	Keyspace  []byte
```

Comment:
not a real problem. The keyspace represent the tenant and all TiDB node in the framework belongs to the same tenant? so we don't need to pass it

Reply:
Yes, but currently I don't want to get the tikv store in subtask executor, so pass it now